payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -759,32 +726,10 @@ data class PaymentMethodCreateParams internal constructor(
             metadata: Map<String, String>? = null
         ): PaymentMethodCreateParams {
             return PaymentMethodCreateParams(
-                type = Type.WeChatPay,
+                type = PaymentMethod.Type.WeChatPay,
                 billingDetails = billingDetails,
                 metadata = metadata
             )
         }
-
```

Comment:
How do you create it now from the compose Map?

Reply:
It was already using the internal constructor, passing in the `paramMap`.